lib/tilelivecopy.js

```diff
@@ -1,25 +1,35 @@
 var url = require('url');
 var tilelive = require('tilelive');
+var tileliveOmivore = require('tilelive-omnivore');
 var TileStatStream = require('tile-stat-stream');
 var queue = require('queue-async');
 var combiner = require('stream-combiner');
 var MigrationStream = require('./migration-stream');
 
 function tilelivecopy(srcUri, s3url, options, callback) {
+  var src_loader = tilelive.load;
   var scheme = 'scanline';
   var protocol = url.parse(srcUri).protocol;
   if (protocol === 'mbtiles:') scheme = 'list';
   if (protocol === 'omnivore:') scheme = 'pyramid';
   options.type = scheme;
+  if (options.bundle === true) {
+    src_loader = function(bundle_uri, cb) {
```

Comment:
@rclark your are right, it's not a 100% clean solution.
It's a shortcut as bundle uris look like this (with 1..n file names separated by `,`):
```
omnivore:///tmp/unpacker.TEST/states-1.geojson,/tmp/unpacker.TEST/states-2.geojson
```
As those uris are new to `tilelive*` modules (and maybe others) I wanted to get it working as fast as possible and avoid going down to rabbit hole.

If we open up bundles to end users this will most certainly have to be revisited.